app/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     // "lib": [],                             /* Specify library files to be included in the compilation. */
     // "allowJs": true,                       /* Allow javascript files to be compiled. */
     // "checkJs": true,                       /* Report errors in .js files. */
-    "jsx": "react" /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */,
+    "jsx": "preserve" /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */,
```

Comment:
There's no need to explicitly import `react` to use JSX when using the [new JSX transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html). Unfortunately we're still using some older packages that _do_ import `react` leading to type check errors.

As the transformation of JSX to JS is handled by Babel we can skip this step here as the TypeScript compiler is only used for type checking.

Reply:
good spot